sdb/command.py

```diff
@@ -27,6 +27,7 @@
 
 import drgn
 
+from sdb import type_canonicalize_name, type_canonical_name
```

Comment:
could you change the above to the following?
`from sdb.target import type_canonicalize_name, type_canonical_name`

If the file is part of the infrastructure (e.g. in `sdb/*.py`, not `sdb/commands`) we want to avoid importing `sdb` as this is our "consumer-facing" API.